e2e/playwright/support/server/default_config.ts

```diff
@@ -529,6 +529,7 @@ const defaultServerConfig: AdminConfig = {
         EnableSharedChannels: false,
         EnableRemoteClusterService: false,
         EnableAppBar: false,
+        PatchPluginsReactDOM: false,
```

Comment:
should we add the type in `packages/types/src/config.ts` too?

Reply:
I added that in the previous commit. I just forgot to add it here in the latest one